### PR TITLE
User Modal

### PR DESCRIPTION
**Pull request by amtang325**
added a user modal pop-up for when a user card is clicked

### CODE REVIEW

**Review by Roshan-D — CHANGES_REQUESTED** [review]
It looks a little funky for me with the descriptions being of variable length, could you fix the css so the button is just stuck to the bottom of each card

Screenshot 2023-07-27 at 8 49 55 AM

File: src/components/UserModal.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          <strong>Description:</strong> {profile.description}
+        </Text>
+        <Divider borderColor="gray.300" borderWidth="2px" />
+            </ModalBody>
+  
+            <ModalFooter>
+              <Button colorScheme='blue'>Ask to Team Up</Button>
+              {/* <Button colorScheme='blue' mr={3} onClick={onClose}>
+                Close
+              </Button> */}
```
Comment (thaneesh-babu) [inline src/components/UserModal.tsx:67]: remove this

**Review by thaneesh-babu — CHANGES_REQUESTED** [review]
Good start! Like said above, there are some styling changes that need to be made:
- The 'see profile' option should be inside the card
- The UserModal should be bigger in size. Although we will be implementing word limit for the description, users would be allowed to write quite a bit and the current space isn't enough for that. Update descriptions of some users to have longer descriptions upto 200 characters

**Review by thaneesh-babu — APPROVED** [review]
lgtm!

**Pull request merged**